src/main/business/applicationFacade.js

```diff
@@ -360,30 +304,29 @@ class ApplicationFacade{
     /**
      * return a list of meta question of the user's course
      * @throws {Error} - If the user is not signed in or does not have the necessary permissions
-     * @return {MetaQuestion[]} all the meta question of the user's course
+     * @return all the meta question of the user's course
      */
-    getAllMetaQuestions(data) {
-        return this.metaQuestionController.getAllMetaQuestions()
+    async getAllMetaQuestions(data) {
+        const businessMQs = await this.metaQuestionController.getAllMetaQuestions();
+        return businessMQs.map(bMQ => this.#mqBusinessToFE(bMQ));
```

Comment:
Cant you just pass this.#mqBusinessToFE ?
If you say it is more readable I will accept

Reply:
do you mean like so?
```
return businessMQs.map(this.#mqBusinessToFE);
```
I didn't know if it was possible but it sounds logical

Reply:
It probably is, just wanted to know...
I whink it is more readable like this anyway...